paz/models/detection/frustum_pointnet.py

```diff
@@ -0,0 +1,370 @@
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.layers import Conv2D, Dense
+from tensorflow.keras.layers import Dropout
+from tensorflow.keras.layers import Input
+from tensorflow.keras.layers import MaxPooling2D
+from tensorflow.keras.models import Model
+
+from examples.frustum_pointnet.dataset_utils import g_mean_size_arr
+from paz.optimization.losses.frustumpointnet_loss import FrustumPointNetLoss
+
+
+def ModelOutputToTensor(output, IntermediateOutputs, NUM_HEADING_BIN=12,
+                        NUM_SIZE_CLUSTER=8):
+    """ Parse batch output to separate tensors (added to IntermediateOutputs)
+    Input:
+        output: TF tensor in shape (B,3+2*NUM_HEADING_BIN+4*NUM_SIZE_CLUSTER)
+        IntermediateOutputs: dict
+    Output:
+        IntermediateOutputs: dict
+
+    """
+    batch_size = output.get_shape()[0]
+    center = tf.slice(output, [0, 0], [-1, 3])
+    IntermediateOutputs['center_boxnet'] = center
```

Comment:
Remove dictionary. It makes it difficult to follow and it seems as if one puts everything there and forgets about it. It would be better to return a list of all the tensors.